integration_test/test_videoroom/test/integration/metadata_test.exs

```diff
@@ -83,14 +82,6 @@ defmodule TestVideoroom.Integration.MetadataTest do
     end
   end
 
-  defp create_room(mustang_options) do
-    # Creating room earlier to avoid error :already_started
-    Task.async(fn ->
-      Stampede.start({RoomMustang, mustang_options}, @browser_options)
-    end)
-    |> Task.await(:infinity)
-  end
-
   defp receive_stats(mustangs_number, pid, acc \\ %{}) do
```

Comment:
Would it be possible to have one `receive_stats` function used in `metadata_test.exs` `simulcast_test.exs` and `basic_stats` or not really?